tests/cephadm/test_os_tuning_profile.py

```diff
@@ -62,16 +83,30 @@ def run(ceph_cluster, **kw):
     mount = "/tmp:/tmp"
     if config.get("command") in ("apply", "re-apply"):
         nodes = config.get("specs", {}).get("placement", {}).get("hosts")
-        hosts = [ceph_cluster.get_nodes()[int(node[-1])].hostname for node in nodes]
```

Comment:
Instead of if..else below code also can be used -
```
        hosts = [ceph_cluster.get_nodes()[int(node[-1])].hostname for node in nodes]
        if "hostx" in nodes:
            hosts = ["NoHost1", "NoHost2"]
```

Reply:
Hi @vamahaja 
We can't able to update code as mentioned.
Reason:
Current condition is updated for negative scenario when host which is not part of cluster is added to tune profile spec.
Code "hosts = [ceph_cluster.get_nodes()[int(node[-1])].hostname for node in nodes]" will fail if we not put inside else statement.

Reply:
+1